content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -191,6 +191,15 @@ and in the logs to monitor the state of the auditing subsystem.
 - `apiserver_audit_error_total` metric contains the total number of events dropped due to an error
   during exporting.
 
+### truncate
+
+Both log and webhook backends support batching. Using log as an example, here's the list of available
```

Comment:
> Using log as an example, here's the list of available
> flags. By default truncate is disabled in both `webhook` and `log`.

As an example, the following is the list of flags available for the log backend:

- f1
- f2
- ...

By default truncate is disabled in both `webhook` and `log`.

* Now the question is how to enable it? *

Reply:
Done.